gitlab-ci-templates/.setup-review-template.yaml

```diff
@@ -9,10 +9,11 @@
     THOAS_SERVICE_SLUG: dev
     REFGET_SERVICE_SLUG: dev
     HELP_DOCS_SERVICE_SLUG: dev
+    TRACK_API_SERVICE_SLUG: dev
 
   before_script:
     - git clone https://gitlab.ebi.ac.uk/ensembl-web/ensembl-k8s-manifests.git
-    - git -C ensembl-k8s-manifests/ checkout wp-k8s-review
+    - git -C ensembl-k8s-manifests/ checkout track-api-integration
```

Comment:
Is this right? You no longer want to use the `wp-k8s-review` branch of `ensembl-k8s-manifests`?

Reply:
After this PR is merged to dev branch, will the template be changed to checkout the dev branch instead of track-api-integration? Or will the template stay the same, but the PR branch would not be removed after merge?

Reply:
If the `.gitlab-ci.yml` file in `ensembl-k8s-manifests` is no longer used, can all the ingress/service templates be checked out from the same branch (e.g. master)?

Reply:
@azangru  We are going to use wp-k8s-review only. The only way to test the integration is by using the track-api-integration branch of Ensembl-k8s-manifest. Which has now been tested so by the time it comes to you for review it will be sorted. :)

Reply:
@veidenberg  The purpose of having it into the master branch was to deploy review app by creating branch of ensembl-k8s-manifest and then push it to ensembl-k8s-manifest. The review app setup has now been moved to Ensembl-client probably we should move it but that will require checking all the respective repos and its integration. or Utilize multi project pipeline to deploy review apps (the ultimate aim).